scripts/install_elan.sh

```diff
@@ -7,5 +7,7 @@ set -o pipefail
 curl -sSfL https://github.com/leanprover/elan/releases/download/v1.4.2/elan-x86_64-unknown-linux-gnu.tar.gz | tar xz
 ./elan-init -y --default-toolchain none
 echo "$HOME/.elan/bin" >> "$GITHUB_PATH"
+lean --version          
+lake --version
```

Comment:
When I tried this branch out on my test repo I get the following error:
```
  /home/runner/work/_actions/oliver-butterley/lean-action/log-info/scripts/install_elan.sh: line 10: lean: command not found
  /home/runner/work/_actions/oliver-butterley/lean-action/log-info/scripts/install_elan.sh: line 11: lake: command not found
```
 [link](https://github.com/austinletson/use-lean-standard-action-with-bare-project/actions/runs/9132706607/job/25114684006#step:3:14) to logs.

I think it may be due to a limitation accessing the `$GITHUB_PATH` within that it just set (see [this](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path) this section in the GitHub docs).

Maybe this:
```suggestion
export PATH="$HOME/.elan/bin":$PATH
lean --version          
lake --version
```